README.rst

```diff
@@ -19,6 +20,12 @@ NewRelic platform. Currently supported backend systems are:
 - Redis
 - Riak
 - uWSGI
+- NagioRelic
+
+  + this is not a backend
+  + it just executes programs (usually bash scripts) in a directory
+  + the program name is used as the metric name
+  + the output of the program must be numeric
```

Comment:
this part should probably go to a different branch, isn't related to the couchbase support part